go/processor/floria/run_context.go

```diff
@@ -74,7 +74,13 @@ func (r runContext) Call(kind tosca.CallKind, parameters tosca.CallParameters) (
 		return tosca.CallResult{}, nil
 	}
 
-	output, isPrecompiled := handlePrecompiledContract(r.blockParameters.Revision, parameters.Input, recipient, parameters.Gas)
+	output, isStatePrecompiled := handleStatePrecompiledContract(
+		r, parameters.Sender, parameters.Recipient, parameters.Input, parameters.Gas)
+	if isStatePrecompiled {
+		return output, nil
+	}
+	output, isPrecompiled := handlePrecompiledContract(
+		r.blockParameters.Revision, parameters.Input, recipient, parameters.Gas)
```

Comment:
A heads-up for when you are implementing warm/cold accounts: If I remember the opera modification in go-ethereum correctly, there is a "bug" in the sense that only real pre-compiled contracts are marked as warm before an execution. The state-precompiled contract is missed there. When you are getting there, please check this carefully.

Reply:
Thank you for the heads up!